test/cases/minicog/minicog_reusable_resources.yml

```diff
@@ -0,0 +1,33 @@
+minicog_reusable_resources:
+# patient resource
+- &patientResource
+  resourceType: Patient
+  name: Luke Skywalker
```

Comment:
this is a very different name format than we typically find in FHIR patient resources - but perhaps that's not germane to your testing?

Reply:
thank you for noticing this.  fixed.